src/mscorlib/src/System/Threading/Mutex.cs

```diff
@@ -224,16 +129,16 @@ private static OpenExistingResult OpenExistingWorker(string name, MutexRights ri
             {
                 throw new ArgumentException(SR.Argument_EmptyName, nameof(name));
             }
+
 #if !PLATFORM_UNIX
-            if (System.IO.Path.MaxPath < name.Length)
+            if (Path.MaxPath < name.Length)
```

Comment:
Should this use the `VerifyNameForCreate` method?

Reply:
The `VerifyNameForCreate` method also checks if name is not null which we don't need in this case but I guess it doesn't really matter performance-wise. I will reuse it here